src/lib/support/logging/CHIPLogging.h

```diff
@@ -396,5 +396,15 @@ bool IsCategoryEnabled(uint8_t category);
  */
 #define ChipLogFormatMessageType "0x%" PRIx8
 
+/**
+ * Logging helpers for cluster ids, so we format them consistently.
+ */
+#define ChipLogFormatClusterId "%" PRIu32
+
+/**
+ * Logging helpers for cluster ids, so we format them consistently.
+ */
+#define ChipLogFormatAttributeId "%" PRIu32
```

Comment:
We already have ChupLogFormatMEI that should be used for cluster and attribute ids, no?

Reply:
That's helpful, switched to using ChipLogFormatMEI